toolchain/check/import_ref.h

```diff
@@ -13,7 +13,7 @@ namespace Carbon::Check {
 
 // Sets the IRs for `ImportIRId::ApiForImpl` and `ImportIRId::Cpp`. Should be
 // called before `AddImportIR` in order to ensure the correct IDs are assigned.
-auto SetApiImportIRs(Context& context, SemIR::ImportIR import_ir) -> void;
+auto SetSpecialImportIRs(Context& context, SemIR::ImportIR import_ir) -> void;
```

Comment:
Sorry, was looking at this again and thinking it's not clear how `import_ir` is used from the signature. Maybe just detail in comments:
 
```suggestion
// Sets the IRs for `ImportIRId::ApiForImpl` from `import_ir`, and
// `ImportIRId::Cpp` from defaults. Should be called before `AddImportIR` in
// order to ensure the correct IDs are assigned.
auto SetSpecialImportIRs(Context& context, SemIR::ImportIR import_ir) -> void;
```

Reply:
Done.